source/_posts/2018-07-11-hassio-images.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "New Hass.io images"
+description: "We release our new images."
+date: 2018-07-11 00:00:00
+date_formatted: "April 22, 2018"
+author: Pascal Vizeli
+author_twitter: pvizeli
+comments: true
+categories: Announcements
+og_image: /images/blog/2018-07-hassio-images/blogpost.png
+---
+
+After 4 month we are happy to announce our new images for Hass.io, based on HassOS.
+
+Note that the descriptions of new System can be get quite technical. Feel free to jump to the [migration steps].
+
+## {% linkable_title What is new %}
+
+This will be a bit technical. 
+
+## {% linkable_title Migration %}
+
+The design of HassOS is diferent to ResinOS. That is the reason why we can't provide a OTA update from old too new System.
```

Comment:
HassOS is fundamentally different from ResinOS, hence, we cannot provide a OTA update for you to update from Hass.io.